lib/Map/LeafletVisualizer.js

```diff
@@ -178,22 +180,37 @@ LeafletGeomVisualizer.prototype.update = function(time) {
     throw new DeveloperError(i18next.t("map.leafletVisualizer.devError3"));
   }
   //>>includeEnd('debug');
+  const bounds = this._leafletScene.map.getBounds();
+  let applyLocalisedAntiMeridianFix = false;
+  let currentBoundsType = 0;
+  if (
+    bounds._southWest.lng > 140 &&
+    (bounds._northEast.lng < -140 || bounds._northEast.lng > 180)
```

Comment:
Needs a comment. Does this detect looking at the antimeridian with the centre being to the east of the antimeridian and being zoomed in?